spec-0001/index.md

```diff
@@ -56,39 +108,17 @@ from scipy.linalg import eig
 eig(...)
 ```
 
-This SPEC proposes a lazy loading mechanism—targeted at libraries—that avoids import slowdowns and brings back explicit submodule exports, but without slowing down imports.
-
-For example, it allows the following behavior:
-
-```python
-import skimage as ski  # cheap operation; does not load submodules
-
-ski.filters  # cheap operation; loads the filters submodule, but not
-             # any of its submodules or functions
-
-ski.filters.gaussian(...)  # loads the file in which gaussian is implemented
-                           # and calls that function
-
-```
-
-This has several advantages:
-
-1. It exposes a **nested namespace that behaves as a flat namespace**.
-   This avoids carefully having to import exactly the right combination of submodules, and allows interactive exploration of the namespace in an interactive terminal.
-
-2. It **avoids having to optimize for import cost**.
-   Currently, developers often move imports inside of functions to avoid slowing down importing their module.
-   Lazy importing makes imports at any depth in the hierarchy cheap.
-
-3. It provides **direct access to submodules**, avoiding local namespace conflicts.
-   Instead of doing `import scipy.linalg as sla` to avoid clobbering a local `linalg`, one can now import each library and access its members directly: `import scipy; scipy.linalg`.
-
-### Usage
-
 Python 3.7, with [PEP 562](https://www.python.org/dev/peps/pep-0562/), introduces the ability to override module `__getattr__` and `__dir__`.
 In combination, these features make it possible to again provide access to submodules, but without incurring performance penalties.
 
-We propose a [utility library](https://pypi.org/project/lazy_loader/) for easily setting up so-called "lazy imports" so that submodules are only loaded upon accessing them.
+### lazy_loader
```

Comment:
If above sentence is put in, the only missing item from this section is the link to github.